source/JointMilitarySymbologyLibraryCS/jmsml/test/Fields_ControlMeasuresAreas.csv

```diff
@@ -0,0 +1,26 @@
+field_name,field_type,field_length,field_alias,nullability,field_domain
+identity,Integer,4,Standard Identity,NULLABLE,Identity
+symbolset,SmallInteger,2,Symbol Set,NULLABLE,
+entity,Integer,4,Control Measures Area,NULLABLE,Control_Measure_Area_Entities
+echelon,Integer,4,echelon,NULLABLE,Echelons
+uniquedesignation,String,50,Unique Designation,NULLABLE,
+additionalinformation,String,50,Additional Information,NULLABLE,
+additionalinformation2,String,50,Additional Information 2,NULLABLE,
+hostile,String,3,Hostile (ENY),NULLABLE,
+type,String,50,Type of Equipment,NULLABLE,
```

Comment:
@csmoore , complete agreement here that the standard needs to be made more consistent.  This is where our interested and attentive SME, @ottenw , comes in.  Bill, can you help shed some light on this.  Does there need to be more than one database storage length for "Type" (amplifier V).  Given your knowledge and access to the various message standards, what is the relationship between message length, display length, and presumably database storage length on an amplifier/attribute like this?  Please and thanks!  :-)